library/Booster/Definition/Ceil.hs

```diff
@@ -156,6 +156,7 @@ computeCeil term@(SymbolApplication symbol _ args)
                 argCeils <- concatMapM computeCeil args
                 pure $ (map Left $ splitBoolPredicates $ Predicate simplified) <> argCeils
 computeCeil (AndTerm l r) = concatMapM computeCeil [l, r]
+computeCeil (OrTerm l r) = concatMapM computeCeil [l, r] -- TODO: Is this too much of an overapproximation? I dont think it should be Ceil(l) \/ Ceil(r)...
```

Comment:
if either L or R is non-bottom, then `Ceil(L) \/ Ceil(R)` should be `Top`. Similarly, if either L or R is non-bottom, then `L \/ R` should be non-bottom, which means `Ceil(L \/ R)` should be Top.

If both L and R are bottom, then `Ceil(L) \/ Ceil(R) == Ceil(L \/ R) == Bottom`

I'm actually more concerned about the line above: `Ceil(L) /\ Ceil(R)` can be `Top` even when `Ceil(L /\ R)` is `Bottom`.